### ISSUE TITLE
Windows 10.

### ISSUE THREAD

**Issue by MichaelPyron** [munawarb/Three-D-Velocity#11]
Does Three-D Velocity work with windows 10? I get an error when I load the program..Narriator tells me that an error has occured, but can't find the error file. any help would be very vuch appreciated..Thanks.

**Comment by blindpilot** [comment]
Munawar    r.
Could you ship me the direct link to grab the TDV program?
I never could get it to work so I quit trying. 
But am curious to try to get it to work again. 
Many Thanks. 
Ron who really would love to find a USB external handled device that would work both with tdv in the world of the 64 bit processor and win 10, but also would work with tdv’s force feedback feature. 
To this day, I still haven’t seen any other hand to ear product with force feedback and or shock feedback written into it like tdv is. 
That’s a huge huge shame. 
Ron
    

From: MichaelPyron 
Sent: Wednesday, April 26, 2017 06:38 PM
To: munawarb/Three-D-Velocity 
Cc: Subscribed 
Subject: [munawarb/Three-D-Velocity] Windows 10. (#11)

Does Three-D Velocity work with windows 10? I get an error when I load the program..Narriator tells me that an error has occured, but can't find the error file. any help would be very vuch appreciated..Thanks.

**Comment by munawarb** [comment]
@MichaelPyron Yes, TDV works on Windows 10. I'm sorry, I can't be of any more help because the error is not something I can work with. However, please check the prerequisites section to make sure you have all that you need.

**Comment by MichaelPyron** [comment]
Hi,

I just found the error I posted about.

I asked if TDV worked with windows 10. I have some friends who tried it, 
and it worked for them..

However, I am getting this error when I launch it..

The text is,


Error log, created with build version 2.23.0.29094:
Error base exception: System.NullReferenceException: Object reference 
not set to an instance of an object.
    at SharpDX.XAudio2.MasteringVoice..ctor(XAudio2 device, Int32 
inputChannels, Int32 inputSampleRate, Int32 deviceIndex)
    at BPCSharedComponent.ExtendedAudio.DSound.initializeOgg() in 
c:\Users\munawar\projects\Three-D-Velocity\shared\BPCSharedComponent\DSound.cs:line 
132
    at TDV.GUI..ctor() in 
c:\Users\munawar\projects\Three-D-Velocity\TDV\GUI.cs:line 81
    at TDV.Common.Main(String[] args) in 
c:\Users\munawar\projects\Three-D-Velocity\TDV\Common.cs:line 238
Error Description: Object reference not set to an instance of an object.
Stack trace:    at SharpDX.XAudio2.MasteringVoice..ctor(XAudio2 device, 
Int32 inputChannels, Int32 inputSampleRate, Int32 deviceIndex)
    at BPCSharedComponent.ExtendedAudio.DSound.initializeOgg() in 
c:\Users\munawar\projects\Three-D-Velocity\shared\BPCSharedComponent\DSound.cs:line 
132
    at TDV.GUI..ctor() in 
c:\Users\munawar\projects\Three-D-Velocity\TDV\GUI.cs:line 81
    at TDV.Common.Main(String[] args) in 
c:\Users\munawar\projects\Three-D-Velocity\TDV\Common.cs:line 238


How can I fix this?
Thanks,
Michael
